src/assets/icon-public.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40">
```

Comment:
This file should also be minified like the `icon-private.svg`

Reply:
Done